dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardUnavailableArtifactTest.java

```diff
@@ -91,7 +91,7 @@ public void testDashboardForRepositoryException() throws TemplateException {
             outputDirectory,
             cache,
             ImmutableMap.of(),
-            new ClassPathResult(LinkedListMultimap.create(), ImmutableList.of()),
+            new ClassPathResult(new AnnotatedClassPath(), ImmutableList.of()),
```

Comment:
Removing obscure LinkedListMultimap in favor of AnnotatedClassPath, which deliver meaning in its name.